imap/http_admin.c

```diff
@@ -91,51 +91,61 @@ static int action_proc(struct transaction_t *txn);
-        { NULL,                 NULL },                 /* CONNECT      */
-        { NULL,                 NULL },                 /* COPY         */
-        { NULL,                 NULL },                 /* DELETE       */
-        { &meth_get,            NULL },                 /* GET          */
-        { &meth_get,            NULL },                 /* HEAD         */
-        { NULL,                 NULL },                 /* LOCK         */
-        { NULL,                 NULL },                 /* MKCALENDAR   */
-        { NULL,                 NULL },                 /* MKCOL        */
-        { NULL,                 NULL },                 /* MOVE         */
-        { &meth_options,        NULL },                 /* OPTIONS      */
-        { NULL,                 NULL },                 /* PATCH        */
-        { NULL,                 NULL },                 /* POST         */
-        { NULL,                 NULL },                 /* PROPFIND     */
-        { NULL,                 NULL },                 /* PROPPATCH    */
-        { NULL,                 NULL },                 /* PUT          */
-        { NULL,                 NULL },                 /* REPORT       */
-        { NULL,                 NULL },                 /* SEARCH       */
-        { &meth_trace,          NULL },                 /* TRACE        */
-        { NULL,                 NULL },                 /* UNBIND       */
-        { NULL,                 NULL }                  /* UNLOCK       */
+      { NULL, NULL },          /* ACL          */
+        { NULL, NULL },          /* BIND         */
+        { NULL, NULL },          /* CONNECT      */
+        { NULL, NULL },          /* COPY         */
+        { NULL, NULL },          /* DELETE       */
+        { &meth_get, NULL },     /* GET          */
+        { &meth_get, NULL },     /* HEAD         */
+        { NULL, NULL },          /* LOCK         */
+        { NULL, NULL },          /* MKCALENDAR   */
+        { NULL, NULL },          /* MKCOL        */
+        { NULL, NULL },          /* MOVE         */
+        { &meth_options, NULL }, /* OPTIONS      */
+        { NULL, NULL },          /* PATCH        */
+        { NULL, NULL },          /* POST         */
+        { NULL, NULL },          /* PROPFIND     */
+        { NULL, NULL },          /* PROPPATCH    */
+        { NULL, NULL },          /* PUT          */
+        { NULL, NULL },          /* REPORT       */
+        { NULL, NULL },          /* SEARCH       */
+        { &meth_trace, NULL },   /* TRACE        */
+        { NULL, NULL },          /* UNBIND       */
+        { NULL, NULL }           /* UNLOCK       */
     }
 };
 
-
 static void admin_init(struct buf *serverinfo __attribute__((unused)))
 {
     namespace_admin.enabled = config_httpmodules & IMAP_ENUM_HTTPMODULES_ADMIN;
 
-    if (!namespace_admin.enabled) return;
+    if (!namespace_admin.enabled) {
+        return;
```

Comment:
I do not like when brackets are inserted after `if` (or `for`) around a single statement - brackets are just extra chars that must be read.